#!/usr/bin/env python
PACKAGE = "wm_admittance"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enableAdmittance", bool_t, 0, "Enable/disable admittance mode", False)
gen.add("verboseMode", bool_t, 0, "Enable/disable verbose mode", False)

exit(gen.generate(PACKAGE, "wm_admittance", "Admittance"))
